projects/core/src/user/model/user.model.ts

```diff
@@ -3,5 +3,4 @@ export interface UserRegisterFormData {
   firstName: string;
   uid: string;
   password: string;
-  titleCode: string;
```

Comment:
shouldn't we leave it here, but as a optional prop? @Xymmer

Reply:
in the first approach we had to remove it from registration at all. Now its reverted